src/marlin/marlin_pc/mod.rs

```diff
@@ -232,11 +124,8 @@ where
                 if enforced_degree_bounds.is_empty() {
                     (None, None)
                 } else {
-                    let mut sorted_enforced_degree_bounds = enforced_degree_bounds.clone();
-                    sorted_enforced_degree_bounds.sort();
-
```

Comment:
Why was this removed? I think we need it, otherwise the `lowest_shifted_power` value will be incorrect.

Reply:
This might be due to incorrect sync with the master branch. Let me take a look